### PR TITLE
New formula: httpdiff

### PR DESCRIPTION
**Pull request by Decors**

### CODE REVIEW

File: httpdiff.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require 'formula'
```
Comment (bfontaine) [inline httpdiff.rb]: Please remove this line.

File: httpdiff.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+require "language/go"
+
+class Httpdiff < Formula
+  homepage "https://github.com/jgrahamc/httpdiff"
```
Comment (bfontaine) [inline httpdiff.rb]: Could you add a short description using `desc "…"`? Thanks!

File: httpdiff.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  def install
+    mkdir_p "#{buildpath}/src/github.com/jgrahamc/"
+    ln_s buildpath, "#{buildpath}/src/github.com/jgrahamc/httpdiff"
+    ENV["GOPATH"] = buildpath
+    ENV.append_path "PATH", "#{ENV["GOPATH"]}/bin"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "go", "build", "-o", "httpdiff"
+    bin.install "httpdiff"
+  end
```
Comment (bfontaine) [inline httpdiff.rb]: Please add a test.

File: httpdiff.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
+require "language/go"
+
+class Httpdiff < Formula
+  homepage "https://github.com/jgrahamc/httpdiff"
+  head "https://github.com/jgrahamc/httpdiff.git"
+  desc "Compare two HTTP(S) responses"
```
Comment (DomT4) [inline httpdiff.rb]: `desc` should go above `homepage`.

Comment (Decors) [inline httpdiff.rb]: Thanks. Fixed it.

**Comment by bfontaine** [comment]
Can someone from @Homebrew/owners give their :+1: / :-1: here please?

File: httpdiff.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  head "https://github.com/jgrahamc/httpdiff.git"
+
+  depends_on "go" => :build
+
+  def install
+    mkdir_p "#{buildpath}/src/github.com/jgrahamc/"
+    ln_s buildpath, "#{buildpath}/src/github.com/jgrahamc/httpdiff"
+    ENV["GOPATH"] = buildpath
+    ENV.append_path "PATH", "#{ENV["GOPATH"]}/bin"
+    Language::Go.stage_deps resources, buildpath/"src"
```
Comment (xu-cheng) [inline httpdiff.rb]: Where are resources?

Comment (DomT4) [inline httpdiff.rb]: `resources` doesn't seem to be defined against anything?

Comment (xu-cheng) [inline httpdiff.rb]: `resources` is defined inside formula. In this case, it's an empty array.

Comment (DomT4) [inline httpdiff.rb]: Yeah, sorry, I meant in this particular case it's not defined against anything because there are no resources used here.

Comment (mikemcquaid) [inline httpdiff.rb]: This still needs fixed.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-head-only/commit/1ef3a68114015a763977a7275401dfcad34d06c1. Thank you for your contribution to Homebrew/head-only, We appreciate it! :smiley_cat:.

It's a nit but new formulae commits should be styled `name (new formula)`, Thanks!

**Comment by bfontaine** [comment]
@DomT4 It’s `name version (new formula)` :wink:

**Comment by DomT4** [comment]
Whoops. @bfontaine is right :wink:

**Pull request closed without merging**